homeassistant/components/switch/wemo.py

```diff
@@ -110,11 +111,30 @@ def device_state_attributes(self):
         if self.insight_params or (self.coffeemaker_mode is not None):
             attr[ATTR_CURRENT_STATE_DETAIL] = self.detail_state
 
+        if self.insight_params:
+            attr['on_latest_time'] = \
```

Comment:
We could put them in ATTR_ constants, but they are probably all going to be renamed and content replaced as part of addressing #3828... more places to change, so should I do it anyway?